app/rdf/query-latest-cube-iri.ts

```diff
@@ -0,0 +1,43 @@
+/** Creates SPARQL query to fetch latest cube iri.
+ * Works for both versioned and unversioned cubes.
+ */
+export const getLatestCubeIriQuery = (cubeIri: string) => {
+  return `PREFIX cube: <https://cube.link/>
+PREFIX schema: <http://schema.org/>
+
+SELECT ?iri WHERE {
```

Comment:
@bprusinowski wouldn't it be useful to also output the `?shape` in this query?
use case: reuse to fetch the shape directly - or potentially `DESCRIBE ?cube ?shape` in a single query...

Reply:
This query is currently used in places where we don't actually need to fetch shape at all (client-side redirect and in `LightCube`) 👍

Reply:
just thinking ahead :smile: - no current need (yet)

Reply:
@bprusinowski won't this come into play once reused for several cases i.e. for Components or the like?
IIUC at least here in the resolver the shape fetching is triggered in a sequence
https://github.com/visualize-admin/visualization-tool/blob/28fedbd216b6ad1fd41196ab7334f181d44513b3/app/graphql/resolvers/rdf.ts#L96-L97